src/scroll-animation.js

```diff
@@ -363,16 +403,47 @@ class AnimatedElement extends Component {
     }
   }
 
+  captureRef(ref) {
+    if (ref) {
+      this.ref = ref.animateRef
+        ? ref.animateRef
+        : ref
+    }
+  }
+
+  renderStructure({ style }) {
+    const { children, classes } = this.props
+
+    const elem = React.Children.map(
+      children,
+      child => React.cloneElement(
+        child,
+        { className: classes, style, ref: this.captureRef.bind(this) }
```

Comment:
Should we be pre-binding `this.captureRef = this.captureRef.bind(this)` in the constructor?